### ISSUE TITLE
Deprecated warning - `path` to replace with `lookup_path` in Embeds

### ISSUE THREAD

**Issue by amandineeap** [ViewComponent/lookbook#491]
### Describe the bug

When checking our application Papertrail warnings , it came up with a deprecated one for `path` to be replaced with `lookup_path` warning.

```
Saw 'DEPRECATION WARNING: path is deprecated and will be removed from Lookbook \d+\.\d+ (use lookup_path instead) (called from block in call at /app/vendor/bundle/ruby/3.2.0/gems/lookbook-2.0.4/app/components/lookbook/embed/inspector/component.html.erb:60)' 4 times
```

Upon investigation, it seems to be coming from the Embed code:

```
.rbenv/versions/3.2.2/lib/ruby/gems/3.2.0/gems/lookbook-2.0.4/app/components/lookbook/embed/inspector/component.html.erb
src: helpers.lookbook_preview_path(target.path, request.query_parameters.merge(lookbook_timestamp: Time.now, lookbook_embed: true)),
```

```
lookbook-2.0.4/app/controllers/lookbook/embeds_controller.rb:56
return redirect_to lookbook_embed_url(scenario ? scenario.path : preview.path, embed_params)
```

### To Reproduce

1. Create a documentation page that contains an embed tag `<lookbook-embed>` 
2. Check for warnings

### Expected behavior

The internal Lookbook Embed code should follow the Lookbook latest conventions and no warning should be shown.

### Version numbers 

Please complete the following information:

* Lookbook: 2.0.4
* ViewComponent: 3.2.0
* Rails: 7.0
* Ruby: 3.2.2

**Comment by allmarkedup** [comment]
Hi @amandineeap, thank you for highlighting this (and for taking time to investigate it too) - I'll fix this up as soon as I have a chance and get it out in the next release 👍

**Comment by allmarkedup** [comment]
I've (finally) just released v2.1.0 that fixes this so I'll close this down now. Thanks for your help!
